### PR TITLE
Language review for page

### PR DESCRIPTION
**Pull request by ruthryi**
Fixes https://github.com/wso2/docs-apim/issues/1440

### CODE REVIEW

File: en/docs/develop/customizations/customizing-the-developer-portal/styling-api-details-info-section.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
 # Styling API Details Info Section
 
-The API details info section can be customized to match with your design needs.
+The "API Details Info" section shown below, can be customized according to your design needs.
 
- ![styling api details info section]({{base_path}}/assets/img/learn/styling-api-details-info-section1.png) 
+ ![styling api details info section]({{base_path}}/assets/img/learn/styling-api-details-info-section1.png)
 
-You can change themes.light.custom.infoBar attributes to change the api detials info section styling. Note these changes will effect the same way to application details info section.
+Edit the attributes in `themes.light.custom.infoBar` to change the styling "API Details Info" section. Note these changes will effect the same way to application details info section.
```
Comment (Mariangela) [inline en/docs/develop/customizations/customizing-the-developer-portal/styling-api-details-info-section.md:7]: "Note these changes will effect the same way to application details info section."

This line indicated that when these configuration changes are applied they get applied to both the API details info section and also the application details info section.

I had an offline discussion with @chanaka3d and he clarified same.

Based on the latter mentioned carryout the following 2 action items -
1. The title of the page needs to indicate that this is applicable to both the API and the Application info details sections

2. You can consider removing the following and make sure that this message is represent right throughout the doc.
"Note these changes will effect the same way to application details info section."

Comment (Mariangela) [inline en/docs/develop/customizations/customizing-the-developer-portal/styling-api-details-info-section.md:7]: Change [1] to [2]

[1] "Edit the attributes in `themes.light.custom.infoBar` to change the styling "API Details Info" section."
[2] Edit the attributes in `themes.light.custom.infoBar` to change the styling "API Details Info" and the "Application Details Info" section.

**Comment by Mariangela** [comment]
@ruthryi , please check if you have already sent these changes to 3.2.0. Remove the SyncPR tag after you are done please.

**Comment by ruthryi** [comment]
The changes have been sent to 3.2.0

**Pull request merged**